libcontainerd/container_windows.go

```diff
@@ -58,6 +59,8 @@ func (ctr *container) start(attachStdio StdioCallback) error {
 		return err
 	}
 
+	defer ctr.debugGCS()
+
```

Comment:
Given that `defer ctr.debugGCS()` is called both on the error path and the normal path, wouldn't it be cleaner to write this (and the above) as:
```
err = ctr.hcsContainer.Start()
defer ctr.debugGCS()
if err != nil {
...
```

Reply:
Ah no. If I deferred the one after start, it wouldn't fire as terminate would have been called by the time the defer was executed, and there wouldn't be a UVM around to debug. Make sense?

Reply:
yes, sorry, missed the subtlety.